packages/e2e-tests/puppeteer/scripts/web3torrent.ts

```diff
@@ -21,7 +21,7 @@ export async function uploadFile(page: Page, handleBudgetPrompt: boolean): Promi
   await page.waitForSelector('input[type=file]');
 
   // Generate a /tmp file with deterministic data for upload testing
-  const fileToUpload = '/tmp/web3torrent-tests-stub';
+  const fileToUpload = `/tmp/web3torrent-tests-stub-${Date.now()}`;
```

Comment:
What if you ensure unique filenames by defining a module-scoped counter, and increment on each call to `uploadFile`?

Reply:
The nice thing about the timestamp is that we don't have to worry about stale browsers hanging around which are already using the `counter` value